lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -25,7 +25,7 @@
 
 """This package provides the TaurusGui class"""
 
-__all__ = ["DockWidgetPanel", "TaurusGui"]
+__all__ = ["DockWidgetPanel", "TaurusGui", "main"]
```

Comment:
It is needed by the generated launcher (the method used by the entry point needs to access to this method)